ji_prospector/prospector/templates/prospector/fanzines/vote.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'base_public.html' %}
+{% load spectre_css %}
+{% load mathfilters %}
+
+{% block title %}Sélection fanzines 2021 {% endblock %}
+{% block subtitle %}{{request.user.get_full_name}}{% endblock %}
+{% block active_menu %}fanzines{% endblock %}
+
+{% block content %}
+<form action="" method="post">
+    <div class="card mb-2" style="width: 30rem;margin: auto; padding: .5em 4em; line-height: 1.6;">
+        <div class="card-header">
+          <div class="card-title h4"><a href="{{obj.image_url}}" target="_blank">{{obj.stand_name}}</a></div>
+        </div>
+        <div class="card-body">
+            <p class="card-text">{{obj.stand_description}}</p>
+            <div style="line-height: 2;">{% if obj.deviant_url %} <a href="{{obj.deviant_url}}" target="_blank" class="card_link">Page Deviant</a> {% endif %}</div>
+            <div style="line-height: 2;">{% if obj.facebook_url %} <a href="{{obj.facebook_url}}" target="_blank" class="card_link">Page Facebook</a> {% endif %}</div>
+            <div style="line-height: 2;">{% if obj.blog_url %} <a href="{{obj.blog_url}}" target="_blank" class="card_link">Blog/Site/Autre</a> {% endif %}</div>
+
+        </div>
+        <div class="card-footer">
+            {% render_form form %}
+            
+            <div class="float-right">
+              <a href="{% url 'prospector:fanzines.vote' pk=obj.pk|sub:1 %}" class="btn">Précédent <i class="icon icon-back"></i></a>
```

Comment:
On ne peut pas assume que les pk des Fanzines sont contigües.
D'ailleurs quand tu click sur précédent dans la première page, ça redirige sur fanzine/vote/0, qui n'existe pas ^^"

Reply:
Alors ouii bon le truc de la première page j'y avais pensé puis j'ai oublié, il faudra le fix. Mais sinon il me semble qu'on *peut* assume que les pk sont contiguës puisqu'elles sont importées en une fois d'un fichiers csv, et que la database leur assigne un truc incrémental non ? Mais c'est vrai que c'est un peu sloppy, et si je le change j'aurais pas forcément besoin d'une nouvelle dépendance sur mathfilters tout ça pour faire un -1... Mais soon not so soon du coup